packages/trader/src/App/Components/Form/TimePicker/__tests__/time-picker.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TimePicker from '../time-picker';
+import moment from 'moment';
+import { mockStore } from '@deriv/stores';
+import TraderProviders from '../../../../../trader-providers';
+
+jest.mock('../dialog', () => jest.fn(() => 'TimePickerDialog'));
+jest.mock('@deriv/components', () => {
+    const original_module = jest.requireActual('@deriv/components');
+    return {
+        ...original_module,
+        Icon: jest.fn(() => <div>Mocked Icon</div>),
+    };
+});
+jest.mock('react-transition-group', () => ({
+    CSSTransition: jest.fn(({ children, ...props }) =>
+        props.in ? <div>{children}</div> : <div>CSSTransitionElement</div>
+    ),
+}));
+
+describe('<TimePicker />', () => {
+    let default_mocked_store: ReturnType<typeof mockStore>;
+    const mocked_props = {
+        end_times: [moment('2023-11-22T12:00:00')],
+        is_nativepicker: false,
+        name: 'testTimePicker',
+        onChange: jest.fn(),
+        placeholder: '',
+        selected_time: '23:45',
+        start_times: [moment('2023-11-21T08:00:00')],
+        validation_errors: ['error', 'error1'],
+    };
+    beforeEach(() => {
+        default_mocked_store = {
+            ...mockStore({
+                ui: {
+                    current_focus: '',
+                    setCurrentFocus: jest.fn(),
+                },
+            }),
+        };
+    });
+    const mockTimePicker = (mock_store: ReturnType<typeof mockStore>) => {
```

Comment:
You are not changing `default_mocked_store` in tests cases, so maybe you don't need parameters in `mockTimePicker` ?